crates/nargo/src/cli/mod.rs

```diff
@@ -127,11 +135,15 @@ pub fn read_inputs_from_file<P: AsRef<Path>>(
     }
 
     let input_string = std::fs::read_to_string(file_path).unwrap();
-    Ok(format.parse(&input_string, abi)?)
+    let mut input_map = format.parse(&input_string, abi)?;
+    let return_value = input_map.remove(MAIN_RETURN_NAME);
+
+    Ok((input_map, return_value))
```

Comment:
Can we add a doc comment to this function explaining the `Option<InputValue>` that is returned is the return value? It is not clear from the type signature or function name alone